Core/tests/ExpressionParser2.cpp

```diff
@@ -959,14 +964,44 @@ TEST_CASE("ExpressionParser2", "[common][events]") {
       node->Visit(validator);
       REQUIRE(validator.GetErrors().size() == 0);
     }
+    SECTION("object function call on group") {
+      auto node =
+          parser.ParseExpression("MyGroup.GetFromBaseExpression()");
+      REQUIRE(node != nullptr);
+      auto &functionNode = dynamic_cast<gd::FunctionCallNode &>(*node);
+      REQUIRE(functionNode.functionName == "GetFromBaseExpression");
+      REQUIRE(functionNode.objectName == "MyGroup");
+      REQUIRE(functionNode.behaviorName == "");
+
+      gd::ExpressionValidator validator(platform, project, layout1, "number");
+      node->Visit(validator);
+      REQUIRE(validator.GetErrors().size() == 0);
+    }
     SECTION("behavior function call") {
       auto node = parser.ParseExpression(
-          "WhateverObject.WhateverBehavior::WhateverFunction()");
+          "MyObject.MyBehavior::GetBehaviorNumberWith1Param(0)");
       REQUIRE(node != nullptr);
       auto &functionNode = dynamic_cast<gd::FunctionCallNode &>(*node);
-      REQUIRE(functionNode.functionName == "WhateverFunction");
-      REQUIRE(functionNode.objectName == "WhateverObject");
-      REQUIRE(functionNode.behaviorName == "WhateverBehavior");
+      REQUIRE(functionNode.functionName == "GetBehaviorNumberWith1Param");
+      REQUIRE(functionNode.objectName == "MyObject");
+      REQUIRE(functionNode.behaviorName == "MyBehavior");
+
+      gd::ExpressionValidator validator(platform, project, layout1, "number");
+      node->Visit(validator);
+      REQUIRE(validator.GetErrors().size() == 0);
+    }
+    SECTION("behavior function call on group") {
+      auto node = parser.ParseExpression(
+          "MyGroup.MyBehavior::GetBehaviorNumberWith1Param(0)");
```

Comment:
Question because I don't remember: do we have tests for failure cases (the object name does not exist as an object or as a group?)